Dockerfile

```diff
@@ -8,4 +7,5 @@ FROM alpine:3.12
 
 COPY --from=0 /go/src/github.com/paypal/load-watcher/bin/load-watcher /bin/load-watcher
 
+RUN ["chmod", "+x", "/bin/load-watcher"]
```

Comment:
It might be a better idea to add this to the Makefile or to the intermediary image where load watcher is built. (One less layer for the actual image)

Reply:
Good point, updated.